common/datavalues/src/types/type_timestamp.rs

```diff
@@ -141,7 +142,7 @@ impl DataType for TimestampType {
     }
 
     fn arrow_type(&self) -> ArrowType {
-        ArrowType::Int64
+        ArrowType::Timestamp(TimeUnit::Microsecond, None)
     }
 
     fn custom_arrow_meta(&self) -> Option<BTreeMap<String, String>> {
```

Comment:
we can remove `custom_arrow_meta`

Reply:
We still need `custom_arrow_meta ` to keep compatibility with old version meta.

Reply:
ok